website/django/data/views.py

```diff
@@ -133,7 +140,7 @@ def index(request):
         count = query.count()
         query = select_page(query, page_size, page_num)
         # call list() now to evaluate the query
-        response = JsonResponse({'count': count, 'synonyms': synonyms_count, 'data': list(query.values(*column))})
+        response = JsonResponse({'count': count, 'deleted_count': deleted_count, 'synonyms': synonyms_count, 'data': list(query.values(*column))})
```

Comment:
if we want to follow the camelcase rule, it could be fixed here by making `'deleted_count' == 'deletedCount'`